zap/src/main/java/org/parosproxy/paros/extension/report/ReportLastScan.java

```diff
@@ -87,15 +88,24 @@ public ReportLastScan() {}
      *     Model model, ReportType reportType)}
      */
     @Deprecated
-    public File generate(String fileName, Model model, String xslFile) throws Exception {
```

Comment:
This change should be reverted (the linked method in the JavaDoc could point to the new method as before).